docs/install.md

```diff
@@ -234,7 +234,7 @@ alternative](https://hub.docker.com/r/linuxserver/code-server).
 
 ## Helm
 
-You can install code-server via [Helm](../ci/helm-chart/README.md).
+You can install code-server via [Helm](https://github.com/cdr/code-server/blob/main/ci/helm-chart/README.md).
```

Comment:
Hmm, that's strange, why does this need to be an absolute link? The relative link does seem to work ok ([see preview](https://coder.com/docs/code-server/v3.11.0/install#helm)) but it links to `https://coder.com/docs/code-server/v3.11.0/ci/helm-chart/README` (missing the .md extension)

I don't know if something else is going on here :thinking:

Reply:
Hey @jawnsy - From what I can tell, content located in `ci` is not rendered on coder.com.

Adding the `.md` extension doesn't fix this, try this link:

https://coder.com/docs/code-server/v3.11.0/ci/helm-chart/README.md

![image](https://user-images.githubusercontent.com/11184711/127432662-785e1600-55c2-486d-886b-e31d0361b1c8.png)

I think it needs to be an absolute link unless we also intend on hosting `ci/helm-chart/README.md` on coder.com/docs from what I see

Reply:
Yeah I think @BrunoQuaresma knows more about this but at least we can fix this issue for now, get people to the right docs and then improve later if needed!

Reply:
Yes, if we want to add it to the docs page, we have to move it to live inside of the docs folder.

Reply:
Sweet! made a followup issue: https://github.com/cdr/code-server/issues/3877